tests/test_blending.py

```diff
@@ -234,3 +234,10 @@ def test_softmax_rgb_blend(self):
 
         self.assertTrue(torch.allclose(dists1.grad, dists2.grad, atol=2e-5))
         self.assertTrue(torch.allclose(zbuf1.grad, zbuf2.grad, atol=2e-5))
+
+    def test_blend_params(self):
+        """Test colour parameter of BlendParams().
+        Assert default value is equal to same passed value (1.0, 1.0, 1.0).
+        """
+        bp1, bp2 = BlendParams(), BlendParams(background_color=(1.0, 1.0, 1.0))
```

Comment:
We need to check if 1) passing in a background color is possible and 2) it is overriding the default. So lets change this test to the following:

```
bp_default = BlendParams()
bp_new = BlendParams(background_color=(0.5, 0.5, 0.5))
self.assertEqual(bp_new.background_color, (0.5, 0.5, 0.5))
self.assertEqual(bp_default.background_color, (1.0, 1.0, 1.0))
```

Reply:
hi @nikhilaravi, I've inserted those code lines.